go/vt/vitessdriver/driver.go

```diff
@@ -139,6 +146,12 @@ type Configuration struct {
 	// This setting has no effect if ConvertDatetime is not set.
 	// Default: UTC
 	DefaultLocation string
+
+	// GRPCDialOptions registers a new vtgate dialer with the provided
+	// dial options.
```

Comment:
Maybe add some more details here about how the protocol string name is going to be registered and what that means if they overwrite the default grpc registration

Reply:
Added more details to the comment.